### PR TITLE
sdg Two Way Frequencies

### PR DESCRIPTION
**Pull request by shadrackkibet**

### CODE REVIEW

**Comment by stevenndungu** [comment]
@dannyparsons this is ready for you

**Comment by shadrackkibet** [comment]
@stevenndungu  i hope that is now fine.

**Comment by stevenndungu** [comment]
Yes, it looks good.

File: instat/sdgTwoWayFrequencies.vb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+' along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+Imports instat.Translations
+Public Class sdgTwoWayFrequencies
+    Public bControlsInitialised As Boolean = False
+    Private Sub sdgTwoWayFrequencies_Load(sender As Object, e As EventArgs) Handles MyBase.Load
+        autoTranslate(Me)
+    End Sub
+
+    Private Sub initialiseDialog()
```
Comment (dannyparsons) [inline instat/sdgTwoWayFrequencies.vb]: Capital letter needed for the start of each word in a sub or function name.

Comment (shadrackkibet) [inline instat/sdgTwoWayFrequencies.vb]: @dannyparsons  that is now changed.

**Pull request merged**